### PR TITLE
Recipes by category

### LINKED ISSUE #3: FE User Story: CategoryDisplay- View
As a user when I click a category from the top of the page,
I want see:

**a Header**

- [x] with the heading, Evans Eats
- [x]  with a plus icon (+) to add recipes

**Recipes**
- [x] all recipes for the selected category

**Footer**
- [x] with a link to the About page
- [x] with a link to the page to delete a recipe

so that I can still see all the options of navigating the site along with all the recipes for that selected category.

### PR DESCRIPTION
**Pull request by jmevans0211**
## What does this PR do?
- Allows for the user to select different categories and see the recipes for that category render on the DOM. This is done by making fetch calls using the `category_id` for the API call to fetch recipes for a single category and updating `recipesSelected` in the store.

### Where should the reviewer start?
- Start on the Home component

### How should this be manually tested?
- Beginning on the homepage, click on each of the categories in the nav. Different recipes should render based on the category clicked.

### Card(s) closed:
Closes #3 

### Screenshots:
evans-eats

### Additional notes/questions:
- Theres a repeated method in Home and CategoryDisplay components. Opportunity to refactor there.
- When deploying to Heroku I ran into this error: `React app error: Failed to construct 'WebSocket': An insecure WebSocket connection may not be initiated from a page loaded over HTTPS` which I solved by changing my `package.json` a bit:
1. ` "scripts": {"start": "serve -s build"}`
2. `react-scripts` version to 3.2.0 from 3.3.0
(seems to be common problem if using the `create-react-app` boiler plate.

### Helpful Resources
-

### CODE REVIEW

File: src/RecipeForm/RecipeForm.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+
+function RecipeForm() {
```
Comment (slkarsh) [inline src/RecipeForm/RecipeForm.js]: can this be an ES6 functional component just for continuity throughout the code base?

File: src/RecipeForm/RecipeForm.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  const [image, setImage] = useState('')
+  const[loading, setLoading] = useState(false)
+
+  const uploadImage = async e => {
+    const files = e.target.files
+    const data = new FormData()
+    data.append('file', files[0])
+    data.append('upload_preset', 'evans_eats')
+    setLoading(true)
+    const response = await fetch(
```
Comment (slkarsh) [inline src/RecipeForm/RecipeForm.js]: should this be an in apiCalls file and imported?

File: src/RecipeForm/RecipeForm.js (hunk 1 above)
Comment (jmevans0211) [reply src/RecipeForm/RecipeForm.js]: Good call. I haven't looked at this component in a while, but definitely yes, should be ES6. Using HOOKS though 🎉🎉🎉

**Comment by slkarsh** [comment]
Current deployment looks great :)

File: src/RecipeForm/RecipeForm.js (hunk 2 above)
Comment (jmevans0211) [reply src/RecipeForm/RecipeForm.js]: 🤔🤔🤔... Probably! Thanks for taking a look at this file too. I think it's a good idea to add it to the `apiCalls` file.

**Pull request merged**